ReactWindows/ReactNative.Shared/Views/Control/ReactControlManager.cs

```diff
@@ -0,0 +1,453 @@
+using Windows.UI.Xaml.Input;
+#else
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Input;
+using System.Windows.Media;
+#endif
+
+namespace ReactNative.Views.ControlView
+{
+    /// <summary>
+    /// View manager for React control instances.
+    /// </summary>
+    public class ReactControlManager : ViewParentManager<ReactControl, ReactControlShadowNode>
+    {
+        private const int FocusCommand = 1;
+        private const int BlurCommand = 2;
+
+        private class TransformDelegateImpl : ITransformControl
+        {
+            private ReactControlManager _manager;
+
+            private ReactControl _control;
+
+            private JArray _transforms;
+
+            public TransformDelegateImpl(ReactControlManager manager, ReactControl control)
+            {
+                _manager = manager;
+                _control = control;
+            }
+
+            public void SetTransform(FrameworkElement view, JArray transforms)
+            {
+                _manager.SetTransform(_control, transforms);
+                _transforms = transforms;
+            }
+
+            public JArray GetTransform(FrameworkElement view)
+            {
+                return _transforms;
+            }
+        }
+
+        /// <summary>
+        /// The name of this view manager. This will be the name used to 
+        /// reference this view manager from JavaScript.
+        /// </summary>
+        public override string  Name
+        {
+            get
+            {
+                return "WindowsControl";
+            }
+        }
+
+        /// <summary>
+        /// The commands.
+        /// </summary>
+        public override IReadOnlyDictionary<string, object> CommandsMap
+        {
+            get
+            {
+                return new Dictionary<string, object>
+                {
+                    { "focus", FocusCommand },
+                    { "blur", BlurCommand },
+                };
+            }
+        }
+
+        /// <summary>
+        /// Exported view constants.
+        /// </summary>
+        public override IReadOnlyDictionary<string, object> ExportedViewConstants
+        {
+            get
+            {
+                return new Dictionary<string, object>
+                {
+                    { "Keys", KeyHelpers.GetKeyConstants() },
+                };
+            }
+        }
+
+        /// <summary>
+        /// Sets whether the view is a tab stop.
+        /// </summary>
+        /// <param name="view">The view instance.</param>
+        /// <param name="isTabStop">
+        /// <code>true</code> if the view is a tab stop, otherwise <code>false</code> (control can't get keyboard focus or accept keyboard input in this case).
+        /// </param>
+        [ReactProp("isTabStop")]
+        public void SetIsTabStop(ReactControl view, bool isTabStop)
+        {
+#if !WINDOWS_UWP
+            view.Focusable = true;
+#endif
```

Comment:
Should this always be set to `true`?

Reply:
I don't know, it's a matter of requirements. It's clear WPF allows for the "focusable, but not through keyboard tabbing" case, something that I'd like to have in the UWP land (and I don't, forcing me to do tricks at a higher level).
We *could* choose a consistent (with UWP) behavior until a WPF only "Focusable" property is exposed. In that case you're right, this has to mimic isTabStop for the time being.

---
In reply to: [159514094](https://github.com/Microsoft/react-native-windows/pull/1546#discussion_r159514094) [](ancestors = 159514094)

Reply:
Let's go for consistency with Windows for now. As long as we don't have a `focusable` (but not tab stop) property, we should revert the `Focusable` value.

---
In reply to: [159515863](https://github.com/Microsoft/react-native-windows/pull/1546#discussion_r159515863) [](ancestors = 159515863,159514094)